docs/Documentation/Drivers/Classic/OutputFormatting.md

```diff
@@ -59,7 +53,7 @@ _type_ = (for binary output files) data type code. Must be one of:
   - `OUT_TYPE_CHAR` = char
   - `*` = use the default type
```

Comment:
Not exactly sure.  I guess they look at the source code. This is legacy behavior from VIC 4.  @tbohn - care to comment?